### PR TITLE
Add camera crosshairs

### PR DESCRIPTION
**Pull request by liamjwang**

### CODE REVIEW

File: src/main/java/org/team1540/robot2018/Robot.java [hunk 1]
```diff
@@ -78,6 +88,28 @@ public void robotInit() {
...
+        //				Imgproc.cvtColor(source, output, Imgproc.COLOR_BGR2GRAY);
+        Point pt1 = new Point(source.width()/2+Tuning.crosshairsSize, source.height()/2);
+        Point pt2 = new Point(source.width()/2-Tuning.crosshairsSize, source.height()/2);
+        Point pt3 = new Point(source.width()/2, source.height()/2+Tuning.crosshairsSize);
+        Point pt4 = new Point(source.width()/2, source.height()/2-Tuning.crosshairsSize);
+        Imgproc.line(source, pt1, pt2, new Scalar(0,255,0), Tuning.crosshairsThicccness);
+        Imgproc.line(source, pt3, pt4, new Scalar(0,255,0), Tuning.crosshairsThicccness);
+        outputStream.putFrame(source);
+      }
+    }).start();
```
Comment (ambercmerrill) [inline src/main/java/org/team1540/robot2018/Robot.java:112]: crosshair code is useful but shouldn't be added to the intake camera

**Review by ambercmerrill — CHANGES_REQUESTED** [review]

File: src/main/java/org/team1540/robot2018/Robot.java (hunk 1 above)
Comment (liamjwang) [reply src/main/java/org/team1540/robot2018/Robot.java:112]: oh right i'm stupid

Comment (liamjwang) [reply src/main/java/org/team1540/robot2018/Robot.java:112]: if we aren't using a turret at all we don't need this

**Comment by liamjwang** [comment]
Turns out this was necessary...

**Review by ambercmerrill — APPROVED** [review]

**Pull request merged**